### DISCUSSION TITLE
CUSTOM DISC ART PLACEHOLDERS

### DISCUSSION THREAD

**Discussion by TT-ReBORN** [TT-ReBORN/Georgia-ReBORN#85] [Show and tell]
Hello community,

as you know Georgia-ReBORN has their own custom designed disc art placeholders which can be used if no disc art 
is available for the album. All users can post and share their custom placeholders with others, this thread is dedicated 
for this purpose.

So go ahead and post some cool ones here, I've already seen some users using their own.

Here is a reminder from the main site:

> If you want you to make your own disc art placeholders or other graphics, you are welcome to download and use
> these resources, respectively templates that contain all Georgia-ReBORN logos plus all cd and vinyl placeholders.
> I would love to see what you have created and I am sure other users feel the same! [Download Resources](https://github.com/TT-ReBORN/Georgia-ReBORN-resources/archive/refs/heads/master.zip).

-TT

**Comment by d3rb** [comment]
Hello TT i make a new Placeholder for you, i have many colored Vinyls at Home an i think okay make a new Placeholder for you.
I think its Looks Cool. :)

i will make more Plaseholder in the near Future, you can Download the Placeholder in my new Git. https://github.com/d3rb/Disc-and-Vinyl-ART-Plaseholder

Have a Nice Time

gretings

![Placeholder1](https://user-images.githubusercontent.com/15198548/199311400-673616a8-8474-45ba-b22d-93672bf7ef74.jpg)
![Placeholder2](https://user-images.githubusercontent.com/15198548/199311458-9ec2c43d-8fcb-4907-a1ff-db1af3070086.jpg)

**Reply by TT-ReBORN** [reply]
Looks cool I've tried it. Btw how did you do it? Scan it yourself with a big scanner!?

**Reply by d3rb** [reply]
i shot this Photo with my mobile Phone "Oppo Find X5 Pro" it has a very good camera 50MP.
The challenge is, that you don't see your own reflection when you take the picture i have take many trys bevor a good shot.

**Comment by d3rb** [comment]
another one :)
"dirty.transparent.marbled.vinyl.png"
![dirty transparent marbled vinyl](https://user-images.githubusercontent.com/15198548/199341650-0b40985a-e389-463e-86fc-5a251dcb29b2.png)

**Reply by TT-ReBORN** [reply]
Looks great, but I personally prefer them without the logos :) 
Or you could just use the Georgia-ReBORN logo only without the type, center it, remove the TT logo 
and copy it to the left side upside down. Or we could make the logo a little more subtle, for example embossed or beveled into the inner disc.

**Reply by d3rb** [reply]
I have make a V2 Version for both.. Centered Labels are perfect in the middel and updated Label a bit.
Okay you will have only the "Dragon Logo" without the ReBorn typ? I will make it

![Vinyl V2](https://user-images.githubusercontent.com/15198548/199487415-f300a56c-23c7-4857-af0a-552961da296e.png)

**Comment by d3rb** [comment]
![Label Logo](https://user-images.githubusercontent.com/15198548/199492282-214c95ee-4390-4e95-81d5-fe5c46dddcd9.jpg)

**Comment by TT-ReBORN** [comment]
This is what I've had in mind, did it quickly with some variations:

Beveled:
![Beveled](https://user-images.githubusercontent.com/79368795/199525428-469d2f69-77ac-4e8c-99e3-08d561948280.png)

Beveled Colored:
![Beveled_Colored](https://user-images.githubusercontent.com/79368795/199525555-e140a596-364c-49e5-9063-d79da46e74eb.png)

Embossed:
![Embossed](https://user-images.githubusercontent.com/79368795/199525628-d3842211-de49-47df-91e2-c83a8e565f85.png)

Embossed Colored:
![Embossed_Colored](https://user-images.githubusercontent.com/79368795/199525716-286bf06e-a89c-4804-94bb-0df64abacc70.png)

If you have Photoshop, here is a template with all 4 logo variations. If you want to change the color on the color variations, you only need to change it in Blending Options > Satin, choose a new color with the color picker. The disc art image should be overall 1400x1400px large, with 10px padding on all sides, that means the disc art is 1380x1380px.

https://cloud.multibits.eu/s/kgP7di7pNSsF8C5

-Tom

**Reply by d3rb** [reply]
you have a PM@hydrogenaudio :)

**Comment by d3rb** [comment]
Hi Tom i dont have Photoshop but it's not a problem i have good sources but i don't know if i can handle them i work with gimp.
In the next step I will create the new labels. do you have the logo as a template for gimp?

Enrico
Preview..

![Vinyl Preview](https://user-images.githubusercontent.com/15198548/199578998-e67be8a9-214b-4682-bd6b-3d750992fb45.png)

**Reply by TT-ReBORN** [reply]
Download the psd file and open it in Gimp, it should also have the layers like in Photoshop.

**Reply by Zephyr0ck** [reply]
> Hi Tom i dont have Photoshop but it's not a problem i have good sources but i don't know if i can handle them i work with gimp. In the next step I will create the new labels. do you have the logo as a template for gimp?
> 
> Enrico Preview..
> 
> ![Vinyl Preview](https://user-images.githubusercontent.com/15198548/199578998-e67be8a9-214b-4682-bd6b-3d750992fb45.png)

Can you share your vinyl scans without design, only the raw vinyl?

**Comment by TT-ReBORN** [comment]
Hey José,

you can also share some of yours with the community if you like and you can also post your screenshots in [this](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/86) thread here. 
You always post good ( like those in the Beta thread ) screenshots of the theme :)

Everyone can post as many as they like, there is no limit.
Thanks!

-Tom

**Comment by d3rb** [comment]
I play a little bit.. what do you think A or B?
I prefer B, "looks like real" Vinyl. A is a little bit cleaner i think.

greetings


![A or B](https://user-images.githubusercontent.com/15198548/201468248-767bbc3b-039d-44eb-8d0e-b8bdf43e8695.png)
![Overview 3 Comp](https://user-images.githubusercontent.com/15198548/201468257-b321862f-b066-4df2-a2fc-811db732c7a3.png)

**Reply by TT-ReBORN** [reply]
Hi Enrico,

you're on the right track! Georgia-ReBORN is a clean theme and actually I would do myself the cleanups of the discs, so I prefer version A! You can also refine it by using the Dodge tool in Photoshop to cleanup the dark stains. One of the best and more advanced tool is the clone stamp tool which is very powerful, if you master this tool you can achieve incredible results. There are also techniques to simulate the grain texture which you could apply with blending in the end, try to google it there are a lot of resources on the internet.

Keep up the good work!

-Tom

**Comment by BooneAllred** [comment]
Is this PSD file still available? All the links 404

**Reply by TT-ReBORN** [reply]
Hi @BooneAllred,

thanks for the interest and mention of the dead link in the first post. 

I have updated it and it will now use this link
https://github.com/TT-ReBORN/Georgia-ReBORN-resources/archive/refs/heads/master.zip
from the official Georgia-ReBORN resources repo:
https://github.com/TT-ReBORN/Georgia-ReBORN-resources

-TT

**Reply by BooneAllred** [reply]
Thank you, also thank you for all your hard work and for providing such an elegant player for us to use.
